src/metabase/models/revision.clj

```diff
@@ -102,8 +102,15 @@
 
 (defn- revision-description-info
   [model prev-revision revision]
-  (let [changes (revision-changes model prev-revision revision)]
-    {:description          (build-sentence changes)
+  (let [changes     (revision-changes model prev-revision revision)
+        description (build-sentence changes)]
+    {:description          (if description
+                             description
+                             ;; HACK: before #30285 we record revision even when there is nothing changed,
+                             ;; so there are cases when revision can comeback as `nil`.
+                             ;; This is a safe guard for us to not display "Crowberto null" as
+                             ;; description on UI
+                             (deferred-tru "made a revision."))
```

Comment:
i'm open to wording suggestions, I thought about "made a change.", but it sounds ambiguous 🤷‍♂️

Reply:
Would `"created a revision"` be nicely symmetrical or would it be misleading? (In other places creating a revision seems to be the wording.)

Should we be explicit and say that they created a revision without any change?

Reply:
yea "create" feel more appropriate.

I changed it to "created a revision with no change."